logs/querysample/tracing.go

```diff
@@ -53,7 +53,10 @@ func ExportQuerySamplesAsTraceSpans(ctx context.Context, server *state.Server, l
 			startTime := sample.OccurredAt.Add(duration)
 			endTime := sample.OccurredAt
 			_, span := tracer.Start(ctx, otelSpanName, trace.WithTimestamp(startTime))
-			span.SetAttributes(attribute.String("url.full", urlToSample(server, grant, sample)))
+			// See https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/database/
+			// however note that "db.postgresql.plan" is non-standard.
+			span.SetAttributes(attribute.String("db.system", "postgresql"))
+			span.SetAttributes(attribute.String("db.postgresql.plan", urlToSample(server, grant, sample)))
```

Comment:
I don't really follow this change. Neither `url.full` nor `db.postgresql.plan` are mentioned in the URL cited from a quick skim, and I don't understand what "plan" means in this context.

Reply:
The problem is that there isn't really a proper place to put information like this - my initial thinking was that `url.full` may be a way to get tracing platforms to actually turn the attribute value into a clickable link, but that does not appear to be the case. It appears `url.full` is intended to represent the URL that an HTTP call from an application goes to, so not really the same as this use case.

I only today discovered that there was the semantic convention for database calls (and how to represent them in spans), and thus the addition of `db.system`. I then thought "how could we represent a query plan within this structure", and since there is some existing cases where there is database-system specific data (https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/database/#call-level-attributes-for-specific-technologies) this seemed like the best name.

Reply:
Got it, thanks for clarifying.